### ISSUE TITLE
Uncaught ReferenceError: Justlazy is not defined

### ISSUE THREAD

**Issue by rogervila** [fhopeman/justlazy#11]
Hello, I'm trying to import the library on my ES6 class, like this:

```js
import Justlazy from 'justlazy';

export default class Image {
    lazy() {
        console.log(Justlazy); // this is working, it shows the object.

        Justlazy.registerLazyLoadByClass('u-lazy');
    }
}
```

When I call `new Image().lazy()` I get this message: `Uncaught ReferenceError: Justlazy is not defined`

What am I doing wrong?

Thank you!

**Comment by fradav** [comment]
@rogervila , see my PR

**Comment by fhopeman** [comment]
thanks! Released in version 1.6.1 right now
